### PR TITLE
Add usage sentence for both building and endpoint file.

### PR DESCRIPTION
**Pull request by lsehn168**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ The CQD Summary Reports dashboard includes a **Tenant Data Upload** page, access
 <a name="BKMKTenantDataFile"> </a>
 
 ### Building data file
-The format of the data file you upload must meet the following to pass the validation check before uploading.
+CQD use Building data file by first derive Subnet column from expanding Network+NetworkRange column, then joining Subnet column to call record’s First Subnet/Second Subnet column to show Building/City/Country/Region… information. The format of the data file you upload must meet the following to pass the validation check before uploading.
```
Comment (nenonix) [inline Teams/turning-on-and-using-call-quality-dashboard.md]:
```suggestion
CQD uses a Building data file derived from a Subnet column (expanding Network+NetworkRange) and the call record’s First Subnet/Second Subnet column to show Building/City/Country/Regional information. The format of the data file you upload must meet the following to pass the validation check before uploading.
```

File: Teams/turning-on-and-using-call-quality-dashboard.md [hunk 2]
```diff
@@ -240,7 +240,7 @@ The format of the data file you upload must meet the following to pass the valid
...
 
 ### Endpoint data file
-The format of the data file you upload must meet the following to pass the validation check before uploading.
+CQD use Endpoint data file by joining its EndpointName column to call record’s First Client Endpoint Name/Second Client Endpoint Name column to show Endpoint Make/Model/Type information. The format of the data file you upload must meet the following to pass the validation check before uploading.
```
Comment (nenonix) [inline Teams/turning-on-and-using-call-quality-dashboard.md]:
```suggestion
CQD uses an Endpoint data file derived from joining the EndpointName column to the call record’s First Client Endpoint Name/Second Client Endpoint Name column to show Endpoint Make/Model/Type information. The format of the data file you upload must meet the following to pass the validation check before uploading.
```

Comment (lsehn168) [reply Teams/turning-on-and-using-call-quality-dashboard.md]: Change use to uses and the in front of call records, prefer don't change other stuff due to:
EndpointName is part of the Endpoint data file, no derive.

File: Teams/turning-on-and-using-call-quality-dashboard.md (hunk 1 above)
Comment (lsehn168) [reply Teams/turning-on-and-using-call-quality-dashboard.md]: changed use to uses, prefer do not change other stuff due to:
1. There are multiple building data files
2. Subnet column is derived from expanding Network+NetworkRange, 2 columns in building data file(s).
3. The column is called region, not regional.

File: Teams/turning-on-and-using-call-quality-dashboard.md (hunk 2 above)
Comment (nenonix) [reply Teams/turning-on-and-using-call-quality-dashboard.md]: Thanks, we edit for language and grammar and  it's completely your call to preserve the technical meaning.

**Comment by LolaJacobsen** [comment]
@siunies I'm merging this one.

**Pull request merged**